package.json

```diff
@@ -8,11 +8,9 @@
   ],
   "scripts": {
     "precommit": "lint-staged && jest && flow check",
-    "lint:md-partial": "remark -u remark-preset-davidtheclark --frail --quiet --no-stdout --output --",
-    "lint:md": "npm run lint:md-partial -- *.md",
     "lint:fix": "eslint . --fix",
-    "lint": "eslint . && npm run lint:md",
-    "format": "prettier --write \"{src/*.js,test/*.js}\"",
+    "lint": "eslint .",
+    "format": "prettier \"**/*.{js,mjs,ts,json,md,yml,yaml}\" --write",
```

Comment:
Let's just include the file formats that we actually have in this list.

Reply:
I updated it to `"format": "prettier \"**/*.{js,json,md,yml,yaml}\" --write",`, I that what you mean?